app/_pricing.pug

```diff
@@ -17,15 +26,15 @@ section#pricing.section.section--dark.section--fullheight
           li WebP and Gif support
           li Image-charts watermark 😢
           li Image-charts.com domain
-          li 
+          li
             a(style="text-decoration: underline", href='//stackoverflow.com/questions/tagged/image-charts',rel="nofollow") Community support
           li &nbsp;
           li &nbsp;
         a.btn.btn__primary(href='//documentation.image-charts.com/') No need to sign up, use it now!
 
       .pricing__item.pricing__item--highlight
         div.pricing__title Enterprise
-        div.pricing__price 49€/month
+        div#pricing__price.pricing__price 529€/year
```

Comment:
vu que c'est un id, il faut être plus précis, ici ça pourrait être `.js-enterprise-price` + `document.querySelector('.js-enterprise-price')`.

Le ".js-*" est une convention habituelle pour préciser sémantiquement que *cette class css là* sera accédée depuis javascript et qu'il ne faut pas l'utiliser comme class css pour du styling (cf: https://medium.freecodecamp.org/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849 )